portable/WizC/PIC18/portmacro.h

```diff
@@ -203,219 +203,220 @@ extern uint16_t usCalcMinStackSize;
+                               *                 clrf    PCLATU,ACCESS \
+                               *             endif \
+                               *             movff   PREINC2,TBLPTRL \
+                               *             movff   PREINC2,TBLPTRH \
+                               *             if __ROMSIZE > 0x8000 \
+                               *                 movff   PREINC2,TBLPTRU \
+                               *             else \
+                               *                 clrf    TBLPTRU,ACCESS \
+                               *             endif \
+                               *             movff   PREINC2,TABLAT \
+                               *             movff   PREINC2,FSR1L \
+                               *             movff   PREINC2,FSR1H \
+                               *             movff   PREINC2,FSR0L \
+                               *             movff   PREINC2,FSR0H \
+                               *             movff   PREINC2,PRODL \
+                               *             movff   PREINC2,PRODH \
+                               *             ; \
+                               *             ; The return from portRESTORE_CONTEXT() depends on \
+                               *             ; the value of ucCriticalNesting. When it is zero, \
+                               *             ; interrupts need to be enabled. This is done via a \
+                               *             ; retfie instruction because we need the \
+                               *             ; interrupt-enabling and the return to the restored \
+                               *             ; task to be uninterruptible. \
+                               *             ; Because bsr, status and W are affected by the test \
+                               *             ; they are restored after the test. \
+                               *             ; \
+                               *             movlb   ucCriticalNesting>>8 \
+                               *             tstfsz  ucCriticalNesting,BANKED \
+                               *             SMARTJUMP _rtos_R4 \
+                               *         _rtos_R3: \
+                               *             movff   PREINC2,BSR \
+                               *             movff   PREINC2,WREG \
+                               *             movff   PREINC2,STATUS \
+                               *             retfie  0       ; Return enabling interrupts \
+                               *         _rtos_R4: \
+                               *             movff   PREINC2,BSR \
+                               *             movff   PREINC2,WREG \
+                               *             movff   PREINC2,STATUS \
+                               *             return  0       ; Return without affecting interrupts \
+                               *         _Pragma("asmend") \
+                               *     } while(0) */
```

Comment:
This whole block got commented out

Reply:
Fixed this in [this commit](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768/commits/d23159490bfb6870c3bdfbf73ebef49ebd3a208e)